com.bambulab.BambuStudio.yml

```diff
@@ -0,0 +1,382 @@
+  - --filesystem=xdg-run/gvfs
+  - --filesystem=/run/media
+  - --filesystem=/media
+  # Allow BambuStudio to talk to other instances
+  - --talk-name=com.bambulab.BambuStudio.InstanceCheck.*
+  - --system-talk-name=org.freedesktop.UDisks2
+  # set dark theme
+  - --env=BAMBU_STUDIO_DARK_THEME=false
+
+modules:
+
+  # xprop, xlib is needed to manipulate the X11 window and set _GTK_THEME_VARIANT dark on X11
+  # and paint the window dark when PRUSA_SLICER_DARK_THEME is true
+  # see: entrypoint & set-dark-theme-variant.py (originated from spotify client flatpak)
+  - name: xprop
+    sources:
+      - type: archive
+        url: http://mirrors.ircam.fr/pub/x.org/individual/app/xprop-1.2.5.tar.gz
+        sha256: b7bf6b6be6cf23e7966a153fc84d5901c14f01ee952fbd9d930aa48e2385d670
+  - name: python-setuptools_scm
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-deps --no-build-isolation --verbose --prefix=${FLATPAK_DEST} .
+    sources:
+      - type: archive
+        url: https://files.pythonhosted.org/packages/57/38/930b1241372a9f266a7df2b184fb9d4f497c2cef2e016b014f82f541fe7c/setuptools_scm-6.0.1.tar.gz
+        sha256: d1925a69cb07e9b29416a275b9fadb009a23c148ace905b2fb220649a6c18e92
+  - name: python-xlib
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-deps --no-build-isolation --verbose --prefix=${FLATPAK_DEST} .
+    sources:
+      - type: archive
+        url: https://files.pythonhosted.org/packages/86/f5/8c0653e5bb54e0cbdfe27bf32d41f27bc4e12faa8742778c17f2a71be2c0/python-xlib-0.33.tar.gz
+        sha256: 55af7906a2c75ce6cb280a584776080602444f75815a7aff4d287bb2d7018b32
+
+  - name: icns
+    sources:
+      - type: git
+        url: https://git.code.sf.net/p/icns/code
+        commit: '194962'
+    cleanup:
+      - '*'
+
+  # For libOSMesa
+  - name: mesa
+    buildsystem: meson
+    config-opts:
+      - -Dosmesa=classic
+      - -Ddri-drivers=[]
+      - -Dgallium-drivers=[]
+      - -Dvulkan-drivers=[]
+      - -Dplatforms=[]
+    build-options:
+      arch:
+        aarch64:
+          config-opts:
+            - -Dlibunwind=disabled
+    cleanup:
+      - /share/drirc.d
+      - /include
+      - "*.a"
+    sources:
+      - type: archive
+        # url: https://archive.mesa3d.org/mesa-20.2.6.tar.xz
+        url: https://ftp.icm.edu.pl/pub/graphics/mesa/mesa-20.2.6.tar.xz
+        sha256: f12ca3c6c622f11cd79ad66b4220f04514fa96f795062fe92a37339ab19885db
+
+  - name: glu
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://ftp.osuosl.org/pub/blfs/conglomeration/glu/glu-9.0.2.tar.xz
+        sha256: 6e7280ff585c6a1d9dfcdf2fca489251634b3377bfc33c29e4002466a38d02d4
+    cleanup:
+      - /include
+      - /lib/*.a
+      - /lib/*.la
+      - /lib/pkgconfig
+
+  - name: glfw
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DBUILD_SHARED_LIBS:BOOL=ON
+      - -DGLFW_USE_WAYLAND=ON
```

Comment:
does it really need "used wayland" when the permissions for the flatpak don't expose wayland?

Reply:
I haven't tested Wayland support at all, as I'm still running on Xorg because of an NVidia GPU, and most of the manifest was copied from PrusaSlicer.